### PR TITLE
update waarg for 9.0

### PR DESCRIPTION
**Pull request by emilypi**

### CODE REVIEW

File: waargonaut.cabal [hunk 1]
```diff
@@ -45,13 +45,8 @@ extra-source-files:
...
 bug-reports:        https://github.com/haskell-waargonaut/waargonaut/issues
 tested-with:
-  GHC ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.4 || ==8.10.3
+  GHC ==8.4.4 || ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.1
 
-custom-setup
-  setup-depends:
-      base           >=4.9 && <5
-    , Cabal
-    , cabal-doctest  >=1   && <1.1
```
Comment (emilypi) [inline waargonaut.cabal:54]: General practice discourages this custom-setup clause. We should use `cabal-docspec` instead.

**Review by gwils — APPROVED** [review]

**Comment by gwils** [comment]
Waarging

File: waargonaut.cabal [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-license:            BSD3
+license:            BSD-3-Clause
 license-file:       LICENSE
 author:             HASKELL-WAARGONAUT @ Data61
 maintainer:
   oᴉ˙ldɟb@uɐǝs, emilypi@cohomolo.gy, george@wils.online, tonymorris+github@gmail.com
 
 copyright:
-  Copyright (C) 2018 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+  Copyright (C) 2018-2021 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
```
Comment (tonymorris) [inline waargonaut.cabal:15]:
```suggestion
  Copyright (C) 2018-2019 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
  Copyright (C) 2020-2021 Someone other than CSIRO (e.g. yourself or System F)
```

**Review by tonymorris — APPROVED** [review]

File: waargonaut.cabal [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+license:            BSD-3-Clause
 license-file:       LICENSE
 author:             HASKELL-WAARGONAUT @ Data61
 maintainer:
   oᴉ˙ldɟb@uɐǝs, emilypi@cohomolo.gy, george@wils.online, tonymorris+github@gmail.com
 
 copyright:
-  Copyright (C) 2018 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+  Copyright (C) 2018-2019 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+  Copyright (C) 2020-2021 Someone other than CSIRO (e.g. yourself or System F)
```
Comment (gwils) [inline waargonaut.cabal:16]:
```suggestion
  Copyright (C) 2020-2021 Sean Chalmers and Emily Pillmore```

File: waargonaut.cabal [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
+license:            BSD-3-Clause
 license-file:       LICENSE
 author:             HASKELL-WAARGONAUT @ Data61
 maintainer:
   oᴉ˙ldɟb@uɐǝs, emilypi@cohomolo.gy, george@wils.online, tonymorris+github@gmail.com
 
 copyright:
-  Copyright (C) 2018 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+  Copyright (C) 2018-2019 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+  Copyright (C) 2020-2021 Sean Chalmers and Emily Pillmore```
```
Comment (emilypi) [inline waargonaut.cabal:16]:
```suggestion
  Copyright (C) 2020-2021 Sean Chalmers and Emily Pillmore
```

**Pull request merged**